imports/ui/components/Modal.jsx

```diff
@@ -1,15 +1,72 @@
+import Button from "@material-ui/core/Button";
+import MaterialModal from "@material-ui/core/Modal";
+import { withStyles } from "@material-ui/core/styles";
 import React, { Component } from "react";
+import FreshForm from "./FreshForm.jsx";
+
+function rand() {
+  return Math.round(Math.random() * 20) - 10;
+}
+
+function getModalStyle() {
+  const top = 50 + rand();
+  const left = 50 + rand();
+
+  return {
+    top: `${top}%`,
+    left: `${left}%`,
+    transform: `translate(-${top}%, -${left}%)`
+  };
+}
 
 class Modal extends Component {
+  state = {
+    open: false
+  };
+
+  handleOpen = () => {
+    this.setState({ open: true });
+  };
+
+  handleClose = () => {
+    this.setState({ open: false });
+  };
+
   render() {
-    // TODO: Create a Form
-    // TODO: Create a Submit Button
+    const { classes } = this.props;
+
     return (
       <div>
-        <h1>Modal</h1>
+        <Button variant="contained" color="secondary" onClick={this.handleOpen}>
+          Add New Post
+        </Button>
```

Comment:
Sorry @YaoLiu93 , I took your task because I needed a button to trigger the modal

Reply:
S'all good, I only wanted to format the button and do initial layout anyway :)